### PR TITLE
TEST: Patched test results for #749

### PR DESCRIPTION
**Pull request by carlwilson**
Patched the integration tests results for #749 submitted by @samalloing thanks.

### CODE REVIEW

File: jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/PdfStream.java [hunk 1]
```diff
@@ -186,8 +205,17 @@ private void extractFilters () throws PdfException
             else {
                 /* Only other allowed value is a string */
                 Filter[] val = new Filter[1];
-                val[0] = new Filter 
-                    (((PdfSimpleObject) filter).getStringValue());
+                if(filter instanceof PdfSimpleObject) {
+                	val[0] = new Filter 
+                            (((PdfSimpleObject) filter).getStringValue());
+                } else {
+					System.out.println("we zijn er");
```
Comment (samalloing) [inline jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/PdfStream.java]: Sorry I left some debug code (System.out.println("we zijn er");). This should be deleted

**Review by samalloing — APPROVED** [review]
It looks fine (except the present debug code. See the inline comment)

Sam

File: jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/PdfStream.java (hunk 1 above)
Comment (carlwilson) [reply jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/PdfStream.java]: That's the issue with bigger PRs and I let the dependency issue take my attention. I'll remove this and re-submit.

**Comment by carlwilson** [comment]
Better @samalloing ?

**Review by samalloing — APPROVED** [review]
thanks!

**Pull request merged**